draft-ietf-tls-tls13.md

```diff
@@ -768,10 +768,10 @@ in the diagram above.
 In the Key Exchange phase, the client sends the ClientHello
 ({{client-hello}}) message, which contains a random nonce
 (ClientHello.random), its offered protocol version, a list of
-symmetric cipher/HKDF hash pairs, some set of Diffie-Hellman key shares (in the
+symmetric cipher/HKDF hash pairs, and its key material as some set of Diffie-Hellman key shares (in the
```

Comment:
It might be more clear to switch to semicolons as list separators for the outer list.